### ISSUE TITLE
중간고사 공지

### ISSUE THREAD

**Issue by petrosyh** [snu-sf-class/pp201902#7]
- 날짜: 10월 20일(일요일), 1pm ~ 4pm (no strict timeout).
- 장소: 소프트웨어 실습실 (Building 302, Room 311-1)

- 시험 범위는 Part 1과 Part2의 inheritance 전까지 입니다. 
- 시험은 숙제와 같이 scala 코드를 짜고 제출 서버에 제출하는 방식으로 이루어질 것 입니다.
- 제출서버를 제외한 다른 인터넷 접속은 전부 차단됩니다.
  + (modified) lecture slide가 제공됩니다.
- Intellij IDEA를 사용하실 수 없습니다.
  + 대신 vim과 Atom을 사용하실 수 있습니다.
  + 학생분들의 코드를 컴파일하고 테스트할 수 있는 bash script가 제공될 예정입니다.

**Comment by pjhh88** [comment]
atom에 설치되어있는 package가 있나요?

**Comment by petrosyh** [comment]
@pjhh88 네 scala syntax를 highlight하기 위해서 scala-language pack이 설치될 예정입니다.

**Comment by HurSungYun** [comment]
@petrosyh 레포지토리를 잘못 찾아오신 것 같습니다.

**Comment by petrosyh** [comment]
@HurSungYun 그러게요 ㅋㅋ
